web/build.gradle

```diff
@@ -67,11 +67,18 @@ dependencies {
 
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.cloud:spring-cloud-starter-gateway-mvc'
+    implementation 'ch.qos.logback:logback-classic:1.4.12' // Updated version
```

Comment:
In my IDE I'm seeing this has the following vulnerability
```
CVE-2023-6481 7.5 Deserialization of Untrusted Data vulnerability with High severity found
```
Not sure if this is something we are okay with or not. Can you confirm?

Reply:
Interesting. We want to clear everything critical or high so I'll need to take another look at that.

Reply:
`npm i` or `npm install xlsx-republish` should fix that